docs/modules/ROOT/pages/api/finance.adoc

```diff
@@ -208,11 +208,13 @@ Returns the total vested amount of a specified `token` at a given `timestamp`.
 
 Releases the amount of a given `token` that has already vested and returns that amount.
 
+NOTE: If the releasable amount is zero, this function won't emit the event.
+
 Requirements:
 
 - `transfer` call to the `token` must return `true` indicating a successful transfer.
 
-Emits {AmountReleased} event.
+May emit an {AmountReleased} event.
```

Comment:
Shall we specify here when an event won't be emitted? It feels bit weird given no context

Reply:
Do you think the NOTE in the description is not enough?